internal/cluster/transition.go

```diff
@@ -489,7 +489,7 @@ func (th *transitionHandler) PostRefreshCluster(reason string) stateswitch.PostT
 		if sCluster.srcState != swag.StringValue(sCluster.cluster.Status) &&
 			funk.ContainsString([]string{statusInfoError, statusInfoInstallingPendingUserAction}, reason) {
 
-			params.metricApi.ClusterInstallationFinished(params.ctx, *updatedCluster.Cluster.Status, sCluster.srcState,
+			params.metricApi.ClusterInstallationFinished(params.ctx, *sCluster.cluster.Status, sCluster.srcState,
```

Comment:
Seems incorrect.  This is the result.  Need the status after update.

Reply:
At this point this is the new status that went into the db. see https://github.com/openshift/assisted-service/blob/master/internal/cluster/transition.go#L483

Reply:
OK, but if the cluster was not updated then it didn't move to the target status.  Actually, nothing happened

Reply:
regardless, moved the metric report after the error check